src/utils/storage/index.ts

```diff
@@ -17,15 +17,17 @@ enum StorageKeysEnum {
   ProfileData = 'profiledata',
   OtherProfileData = 'otherprofiledata',
   LastPromptNotification = 'lastPromptNotification',
-  LastSelectedMenu = 'lastSelectedMenu'
+  LastSelectedMenu = 'lastSelectedMenu',
+  BlockedStatus = 'blockedStatus',
+  BlockingStatus = 'blockingStatus'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `BlockedStatus` and `BlockingStatus` to the `StorageKeysEnum` is a good way to manage these new keys. However, it's important to ensure that these keys are used consistently throughout the application to avoid any potential issues with data inconsistency or retrieval errors.

```diff
   LastSelectedMenu = 'lastSelectedMenu',
+  BlockedStatus = 'blockedStatus',
+  BlockingStatus = 'blockingStatus'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->